st2reactor/st2reactor/rules/enforcer.py

```diff
@@ -16,21 +16,26 @@
 import json
 
 from st2common import log as logging
+from st2common.constants import action as action_constants
 from st2common.constants.trace import TRACE_CONTEXT
-from st2common.util import reference
-from st2common.util import action_db as action_db_util
-from st2reactor.rules.datatransform import get_transformer
-from st2common.services import action as action_service
-from st2common.services import trace as trace_service
 from st2common.models.api.trace import TraceContext
 from st2common.models.db.liveaction import LiveActionDB
+from st2common.models.db.rule_enforcement import RuleEnforcementDB
 from st2common.models.utils import action_param_utils
-from st2common.constants import action as action_constants
 from st2common.models.api.auth import get_system_username
+from st2common.persistence.rule_enforcement import RuleEnforcement
+from st2common.services import action as action_service
+from st2common.services import trace as trace_service
+from st2common.util import reference
+from st2common.util import action_db as action_db_util
+from st2reactor.rules.datatransform import get_transformer
 
 
 LOG = logging.getLogger('st2reactor.ruleenforcement.enforce')
 
+EXEC_KICKED_OFF_STATES = [action_constants.LIVEACTION_STATUS_SCHEDULED,
```

Comment:
Maybe. I don't want to expose this as the golden standard for EXEC_KICKED_OFF. In this context, this makes sense.